app/middleware/locationValidator.js

```diff
@@ -2,6 +2,13 @@ const log = require('../lib/logger');
 const renderer = require('../middleware/renderer');
 const locationValidator = require('../lib/locationValidator');
 
+function setLocationLabel(res, location) {
+  if (location) {
+    // eslint-disable-next-line no-param-reassign
+    res.locals.locationLabel = 'Enter a valid postcode';
```

Comment:
In response to the now deleted comment about why that text is there (a good question and something I should have commented on when I did it):

I've done it because it is variable (gets overridden when an invalid location has been entered) and I didn't want any logic for determining which text to use in the view.

Good idea to set the page title in middleware.